passflip/arguments.py

```diff
@@ -15,10 +15,15 @@ def init_arguments(self):
             help="prompt for password & salt twice."
         )
         self.add_argument(
+            "-m",
+            "--multiple",
+            action="store_true",
+            help="prompt forever until input is blank"
+        )
+        self.add_argument(
```

Comment:
Oh right, we're talking about the inheritance now? Yeah, nobody's disagreeing with you.